src/Form.tsx

```diff
@@ -91,7 +75,7 @@ const FormImpl: Form = (React.forwardRef(
         className={classNames(
           className,
           validated && 'was-validated',
-          inline && `${bsPrefix}-inline`,
+          bsPrefix,
```

Comment:
I think we can drop `bsPrefix` here?  I looked at the v5 docs and there is no `.form` class usage.

Reply:
Hmm, why was the `form` class dropped? Also, if it was dropped, should we just drop our Form component entirely?

I have noticed that a lot of other component libraries (e.g. MUI and Chakrai-UI) don't have an overall Form component, so I'm curious if we should carry that trend over to our library as well.

Reply:
Took a closer look at bootstrap docs and it looks like there wasn't a `.form` class in 4.0 either.  I've seen usage of `.form-inline` and `.was-validated`.  

https://getbootstrap.com/docs/4.0/components/forms/

So in v5 with inline being dropped, this just leaves `.was-invalidated` to handle.  I guess it's nice to have Form around even if it's to handle 1 class... unsure :)

Reply:
Either way the diffs for dropping support for the Form component would warrant it to be in a separate PR altogether, so we should open a new issue and discuss dropping support there.

In the meantime I'll drop `bsPrefix` and get this merged 👍